src/ameba.cr

```diff
@@ -40,4 +40,8 @@ module Ameba
   def run(config = Config.load)
     Runner.new(config).run
   end
+
+  def self.ecr_supported? : Bool
+    {{ compare_versions(Crystal::VERSION, "1.15.0") >= 0 }}
+  end
```

Comment:
I suggest moving this to config, but may be subjective. @Sija wdyt ?

Reply:
My thought was if they didn't want to lint ECR files, they can add `**/*.ecr` to `Excluded`